internal/ui/termstatus/status.go

```diff
@@ -95,7 +95,7 @@ func (t *Terminal) run(ctx context.Context) {
 	for {
 		select {
 		case <-ctx.Done():
-			if IsProcessBackground() {
+			if IsProcessBackground(t.fd) {
```

Comment:
This is equivalent to stdout. Why not just replace stdin with stdout in `IsProcessBackground`?

Reply:
The Terminal type is designed to work with any Writer that has an Fd method and never mentions stdout. If for whatever reason it's changed to use a different fd, IsProcessBackground would break.